packages/sdk/src/types/ParagraphStyleTypes.ts

```diff
@@ -75,6 +78,18 @@ export type ParagraphStyleUpdate = {
     color: {
         value: ColorUsageUpdate;
     };
+    fillColorApplied: {
+        value: boolean;
+    };
+    strokeColor: {
+        value: ColorUsageUpdate;
+    };
+    strokeColorApplied: {
+        value: boolean;
+    };
+    strokeWidth: {
+        value: string;
```

Comment:
What was the constraint on `strokeWidth` and points i'm confused, we default the measurement type to points on the engine side when not defined in the string itself but this is just a regular unit otherwise 🤔

Reply:
For updating we allow strings so the end user can write formulas or have the 'pt' attached etc. We always send points back

Reply:
I assumed string was by default considered unitValue by SDK but if that's not the case then engine side dtos need to be updated as they are sending double right now.